org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -569,12 +570,15 @@ private GameObject FindInteractorGroupObject(XRBaseInteractor interactor)
             {
                 interactorGroupObject = modeManagedInteractor.ModeManagedRoot;
 
+                Assert.IsNotNull(interactorGroupObject, $"Interactor {interactor.name} ({interactor.GetType().Name}) is not managed by any interactor group. " + Environment.NewLine +
+                        $"Please ensure that the interactor implements IModeManagedInteractor, has a ModeManagedRoot field, and that ModeManagedRoot is set to the parent GameObject.");
```

Comment:
Don't put this message here...legacy uses may still have GetModeManagedController returning non-null.

I still think we should put an assert in the "RegisterInteractor() call if `FindInteractorGroupObject` returns null.   To avoid double asserts when called by `Register` and `Unregister`

Reply:
I see your point, moved in fda69688e2872370133dff26eb7d5e55b6272fab, resolving.